package.json fix? (#378)
solid/solid-panes: README.md

contacts (A VCARD Address Book, Group, Individual, Organization) can be handled by the one contact
pane. Any other pane which wants to deal with contacts can just use the pane within its own user interface.

> If you have trouble seeing syntax highlighting for SPARQL, Turtle, TriG, N-Triples, N-Quads, etc...
> Try installing an extension! For VSCode, there are many, but a great one is RDFox Syntax Highlighting. 

Reviewer: ```suggestion
If you have trouble seeing syntax highlighting for SPARQL, Turtle, TriG, N-Triples, N-Quads, etc.,
try installing an extension! For VSCode, there are many; one that we've used is 
[RDFox Syntax Highlighting](https://marketplace.visualstudio.com/items?itemName=rdfox.rdfox-rdf). 
```
